adagrams/game.py

```diff
@@ -1,13 +1,126 @@
 from random import randint
 
 def draw_letters():
-    pass
+    tiles = [
+    'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 'A', 
+    'B', 'B',
+    'C', 'C',
+    'D', 'D', 'D', 'D', 
+    'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 'E', 
+    'F', 'F', 
+    'G', 'G', 'G', 
+    'H', 'H',  
+    'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 'I', 
+    'J',  
+    'K', 
+    'L', 'L', 'L', 'L', 
+    'M', 'M', 
+    'N', 'N', 'N', 'N', 'N', 'N', 
+    'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 
+    'P', 'P', 
+    'Q', 
+    'R', 'R', 'R', 'R', 'R', 'R', 
+    'S', 'S', 'S', 'S', 
+    'T', 'T', 'T', 'T', 'T', 'T',
+    'U', 'U', 'U', 'U', 
+    'V', 'V', 
+    'W', 'W', 
+    'X', 
+    'Y', 'Y', 
+    'Z'
+    ]
+    selected_index = []
+    picked_letters = []
+    for i in range(0, 10):
```

Comment:
Notice that you declare the looping variable `i`, but don't use it in the body of your for loop. When we write a for loop, but don't need the looping variable, we should use `_` instead.
```suggestion
    for _ in range(0, 10):
```